RichText: Fix outdated docs (#10668)
WordPress/gutenberg: packages/editor/src/components/rich-text/README.md

# `RichText`

Render a rich [`contenteditable` input](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Editable_content), providing users with the option to format the content.

## Properties

### `value: String`

> *Required.* HTML string to make editable. The rendered HTML should be valid, and valid with respect to the `tagName` and `multiline` property.

Reviewer: I think it more means that you should not use any tags that cannot be children of the `tagName` you provide. The example you're giving might confuse people in that they'd have to use multiline in combination with `tagName="p"`? Maybe we can just leave this away.

> *Required.* HTML string to make editable. The rendered HTML should be valid, and valid with respect to the `tagName` and `multiline` property.

Reviewer: ```suggestion
*Required.* HTML string to make editable. The HTML should be valid, and valid inside the `tagName`, if provided.
```
